src/main/java/frc/robot/constants/AutoConstants.java

```diff
@@ -11,69 +13,74 @@ public class AutoConstants {
 	public static final double SOURCE_X_OFFSET = 0;
 	public static final double SOURCE_Y_OFFSET = 0;
 
-	public static final int BLUE_L_STATION_ID = 1;
-	public static final int BLUE_R_STATION_ID = 1;
+	public static final int BLUE_L_STATION_ID = 12;
+	public static final int BLUE_R_STATION_ID = 13;
 	public static final int RED_L_STATION_ID = 2;
-	public static final int RED_R_STATION_ID = 2;
+	public static final int RED_R_STATION_ID = 1;
 
 	/* -- ALL REEF SPECIFIC CONSTANTS -- */
 	public static final double REEF_X_L_TAG_OFFSET = -0.5;
 	public static final double REEF_X_R_TAG_OFFSET = 0.5;
 	public static final double REEF_Y_L_TAG_OFFSET = 0;
 	public static final double REEF_Y_R_TAG_OFFSET = 0;
 
-	public static final int R_REEF_1_TAG_ID = 9;
-	public static final int R_REEF_2_TAG_ID = 9;
-	public static final int R_REEF_3_TAG_ID = 9;
-	public static final int R_REEF_4_TAG_ID = 9;
-	public static final int R_REEF_5_TAG_ID = 9;
+	public static final int R_REEF_1_TAG_ID = 10;
```

Comment:
Consider defining an integer enum for these. https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html